src/backend/distributed/sql/udfs/citus_backend_gpid/latest.sql

```diff
@@ -1,8 +1,9 @@
-CREATE FUNCTION pg_catalog.citus_backend_gpid()
+DROP FUNCTION pg_catalog.citus_backend_gpid;
+CREATE FUNCTION pg_catalog.citus_backend_gpid(pid int default NULL)
     RETURNS BIGINT
-    LANGUAGE C STRICT
+    LANGUAGE C
```

Comment:
We imitated `pg_backend_pid` to create `citus_backend_gpid`. I think its semantics is very simple: Just return the gpid. 

Instead of changing this, can't we introduce another function for this purpose?

Reply:
Changed it's usage to `citus_calculate_gpid(citus_backend_nodeid(), pid)`